### PR TITLE
SPOI-12412 SPOI-12425 #resolve documentation for drools and superset

### PR DESCRIPTION
**Pull request by mohini-sri90**

### CODE REVIEW

**Comment by jessicawangdt** [comment]
"Click Add from Repository button. The Artifacts page is displayed and the schema that was created in DT RTS for the application configuration is listed there. "

just tried it, didn't work, nothing is listed:
screen shot 2018-02-16 at 6 50 51 pm

while the configuration package I tested with contains two jar artifacts:
screen shot 2018-02-16 at 6 51 28 pm
http://node35.morado.com:9090/static/#/dev/configurations/dtadmin/IP2-FPA/0.0.1

**Comment by jessicawangdt** [comment]
@mohini-sri90 : 
I just watched the video:
https://drive.google.com/file/d/17ihYewPxvTZd9jUNpHXPLPO1FkTtcGTe/view
where I did see the popup from "Add from repository" does contain the JAR artifact in the configPkg. 
screen shot 2018-02-16 at 7 03 40 pm

However, when I followed the same step on QA's web site, the popup from "Add from repository" still does NOT contain JAR artifact in that configuration package like I mentioned in my previous comment:

http://node35.morado.com:9090/static/#/dev/configurations/dtadmin/IP2-FPA/0.0.1

File: docs/omni_channel_fraud_app.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The following table lists the various processing phases of the incoming data tup
 | --- | --- | --- |
 | 1 | Transaction Ingestion | Data from all your input sources is pushed onto a message bus such as Apache kafka. The application leverages platform capabilities to ingest data from Kafka message bus to rest of its pipeline. Data tuples are filtered and transformed. The following operators are involved during Transaction ingestion: *Kafka Input Operator*, *Filter Operator*, *Transformation Operator*.|
 | 2 | Transaction Enrichment | Data is enriched with the lookup information such as user profile data, card data, geo data, store profile data etc.The following operators are involved in the transaction enrichment phase: *Enricher*, *Geo Data*, *Customer Data*, *Card Data*, *Product Data*.|
-| 3 | Fraud Rule Execution | Fraud detection rules are applied to the enriched data using drools as CEP engine. The following operators are involved in the Fraud Rule Execution phase: *Fraud Rule Executor*.|
+| 3 | Fraud Rule Execution | Fraud detection rules are applied to the enriched data using drools as CEP Workbench. The following operators are involved in the Fraud Rule Execution phase: *Fraud Rule Executor*.|
```
Comment (DT-Priyanka) [inline docs/omni_channel_fraud_app.md]: The change is not right. The rules are applied / executed using CEP engine and not workbench.

**Comment by DT-Priyanka** [comment]
reviewed drools wb doc contents.

**Comment by jessicawangdt** [comment]
maybe changes:

"Go to your Drools project and click Create New Asset . A file is created which gets listed in the project."

to:

"Go to your Drools project and click Create New Asset, select "DRL file" from dropdown . A file is created which gets listed in the project."

File: docs/cep_workbench.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Introduction
+
+CEP Workbench is a [DataTorrent RTS Service](services/#overview) that allows users to govern Drools Workbench assets. The service provides you with the capability to change the application functionality, using drools-based rules.
```
[the remark this answers is not in the document]
Comment (sashadt) [reply docs/cep_workbench.md]: @mohini-sri90 Let's say that "CEP Workbench is a customized Drools Workbench implementation" or something along those lines.  I think we should avoid link at this level.  Maybe somewhere further down where specifics of rule creation are needed.

File: docs/omni_channel_fraud_app.md (hunk 1 above)
Comment (sashadt) [reply docs/omni_channel_fraud_app.md]: That is correct.  In execution flow it's "engine"

**Comment by jessicawangdt** [comment]
@mohini-sri90 : 

"On the upper right side, click Build & Deploy button."  

Did "Build & Deploy" ever succeed when you tested?
It always failed when I tested it:

screen shot 2018-02-19 at 2 04 40 pm

File: docs/cep_workbench.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+The CEP Workbench service is pre-packaged with applications like Omni-channel Fraud Prevention and Account Takeover Prevention. Using this service, you can configure customized rules for an application in DT RTS. From the DT RTS console, you can access CEP Workbench service, create the customized rules, and then apply these rules to the application configuration before you launch the application.
+
+# Pre-requisites
+
+Install Docker (Version 1.9.1 or greater). You can install Docker during or after RTS installation and specify the docker host in the installation wizard. For more details, please refer to [Docker Configuration](services/#docker-configuration)
```
[the remark this answers is not in the document]
Comment (mohini-sri90) [reply docs/cep_workbench.md]: The page is in another PR. But we have verified the link

**Pull request merged**